docs/Versioning.md

```diff
@@ -26,16 +26,16 @@ For more details on ONNX Release versions, see [this page](https://github.com/on
 
 | ONNX Runtime release version | ONNX release version | ONNX opset version | ONNX ML opset version | Supported ONNX IR version | [WinML compatibility](https://docs.microsoft.com/en-us/windows/ai/windows-ml/)|
 |------------------------------|--------------------|--------------------|----------------------|------------------|------------------|
-| 1.4.0 | **1.7** down to 1.2 | 12 | 2 | 6 | -- |
-| 1.3.1 | **1.7** down to 1.2 | 12 | 2 | 6 | -- |
-| 1.3.0 | **1.7** down to 1.2 | 12 | 2 | 6 | -- |
-| 1.2.0<br>1.1.2<br>1.1.1<br>1.1.0 | **1.6** down to 1.2 | 11 | 2 | 6 | -- |
-| 1.0.0 | **1.6** down to 1.2 | 11 | 2 | 6 | -- |
-| 0.5.0 | **1.5** down to 1.2 | 10 | 1 | 5 | -- |
-| 0.4.0 | **1.5** down to 1.2 | 10 | 1 | 5 | -- |
-| 0.3.1<br>0.3.0 | **1.4** down to 1.2 | 9 | 1 | 3 | -- |
-| 0.2.1<br>0.2.0 | **1.3** down to 1.2 | 8 | 1 | 3 | 1903 (19H1)+ |
-| 0.1.5<br>0.1.4 | **1.3** down to 1.2 | 8 | 1 | 3 | 1809 (RS5)+ |
+| 1.4.0 | **1.7** down to 1.2 | 12 | 2 | 6 | WinAI 1.4+ |
```

Comment:
Can we change it to "Windows AI 1.4+" which will match the docs (which i am updating)

Reply:
ok